build.gclient

```diff
@@ -1,42 +1,63 @@
 # TODO(pphaneuf): Make this be good.
 
-.PHONY: configure-ct openssl protobuf libevent libevhtp gflags glog ldns sqlite3 leveldb json-c tcmalloc
+INSTALL_DIR?=$(shell pwd)/install
+export INSTALL_DIR
+
+PHONY: libunwind tcmalloc openssl protobuf libevent libevhtp gflags glog ldns sqlite3 leveldb json-c configure-ct
 
 all: configure-ct
 
-tcmalloc:
+_libunwind:
+	if [ -d libunwind ]; then \
+		$(MAKE) -C libunwind -f ../certificate-transparency/build/Makefile.unwind; \
+		(cd libunwind && git checkout --); \
+	fi
+
+_tcmalloc:
 	$(MAKE) -C tcmalloc -f ../certificate-transparency/build/Makefile.tcmalloc
+	(cd tcmalloc && git checkout --)
```

Comment:
No need for parenthesis here? They *are* needed for the `_libunwind` command, though, I agree.